File: LV 1/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    zero_count = len([lotto for lotto in lottos if lotto == 0])
    current_count = len([lottos[i] for i in range(len(lottos)) if lottos[i] in win_nums])
    
    high_rank = min(6, 7 - (current_count + zero_count))
    low_rank = min(6, 7 - (current_count))

    answer = [high_rank, low_rank]
    return answer

File: LV 1/JS/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    var answer = [];
    var zero_count = 0;
    var correct_count = 0;
    
    for (let lotto of lottos) {
        if (lotto == 0) {
            zero_count++;
        } else {
            if (win_nums.indexOf(lotto) != -1) {
                correct_count++;            
            }
        }
    }
    
    answer[0] = Math.min(6, 7 - (zero_count + correct_count));
    answer[1] = Math.min(6, 7 - correct_count);
    
    return answer;
}
